sulis_state/src/item_list.rs

```diff
@@ -108,6 +108,14 @@ impl ItemList {
         Some(self.items.remove(index))
     }
 
+    pub fn identify(&mut self, index: usize) -> Option<&ItemState> {

```

Comment:
This approach that you have taken does not store the identified status in the save.  So identifying an item does not persist when saving.  (You need to re-identify every time you save and load).

Reply:
should be resolved after last update